src/components/forms/GranteeFinanceForm.tsx

```diff
@@ -328,6 +333,80 @@ export const GranteeFinanceForm: FC = () => {
                   </FormControl>
                 )}
               />
+
+              <Box display={isL2Selected ? 'block' : 'none'}>
+                <Fade in={isL2Selected} delay={0.25}>
+                  <Controller
```

Comment:
<img width="1194" alt="image" src="https://github.com/ethereum/esp-website/assets/54227730/73fdba92-44ff-4dd1-9417-e5e5e7208668">

@pettinarip This field ("Which Layer 2 Network ?") doesn't seem to be required even when "Yes" is checked for "Layer 2 Payment"... I'm having a hard time testing though since I'm being blocked by the security ID, but it's not giving me a warning about this field being empty when I get to the end.

Reply:
(I should say both of these conditional fields, not just the first one)

Reply:
So, these are both optional fields. They are not required. Not sure if I'm answering your concern.

Reply:
> If you select "l2 payment" then "l2 network" is a required field.
> If you select "Other - as l2 network" then "l2 network other" field is required.

This was my question... Seemed like it should be required conditionally